pkg/crt/repocontext.go

```diff
@@ -47,13 +57,46 @@ func GetRepoContext(dir string) (RepoContext, error) {
 		return RepoContext{}, err
 	}
 
+	sourceHash, err := getSourceHash(dir, ignoreDirs)
+	if err != nil {
+		return RepoContext{}, err // blah
+	}
+
+	var moduleName string
```

Comment:
Would it be better to extract this value using [this](https://pkg.go.dev/golang.org/x/mod/modfile#Module) instead of a custom parser?

Reply:
Ah, yes, thanks, I hadn't seen that package, I'll update to use it.

Reply:
Done